README.md

```diff
@@ -1,10 +1,10 @@
-# boring
+# `boring`
```

Comment:
We don't normally format titles in code, as in sanitize-html. It reveals that the thing  might not have been given a human n the title of the thing. 

[Chalk](https://www.npmjs.com/package/chalk) is one example. It's been given the readable name of "Chalk," which looks like a normal English proper noun. webpack, is officially lowercase (as boring was), and is also the command, but isn't formatted as code.

TLDR; we should leave those as it was or capitalize them outside of actual code examples.